data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,450 @@ mission "Unfettered returning home"
+			action
+				fail
+			`As you take to join your fleet in orbit, you receive a hail from the Unfettered ships. "If you can't even count how many ships you have, you're not worthy of our time, monkey!"`
+	on disabled
+		fail
+		conversation
+			`The enemy ships hail you after disabling your ship. "Thought you could get away from us so easily? You better give us a good bribe to land on our worlds and return that ship, before we value blowing you up more than preserving it!"`
+				to display
+					has "unfettered: fled with the relic"
+				decline
+			`After your ship is disabled, one of the Unfettered ships hails you. "You have lost the challenge, but we didn't expect much from a human like you. We will repair your ship for a fair bribe, but you better land at one of our worlds and return it to us with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
+			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us. Maybe someday you will prove yourself otherwise."`
+	to fail
+		not "flagship landed"
+		has "unfettered: fled with the relic"
+	on fail
+		"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+		clear "old reputation: Hai (Unfettered)"
+		dialog `The Unfettered hail you. "You failed to complete the duel, and as such, are no longer authorized to land on our worlds. You will not enjoy unwarranted respect any longer! It would be in your best interest to return the ship to us on Darkcloak, but any of our worlds will do."`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+			"old reputation: Hai (Unfettered)" >= -2000
+		personality disables heroic marked launching plunders staying target unconstrained
+		ship "Sea Scorpion (Ionic)" "ThunderStruck"
+		on kill
+			dialog `You receive a hail from Mera, who sounds quite angry. "Destroying the ship is either incompetence or evil intent, and we will not have either!"`
+			fail
+		conversation
+			`You receive a message from the Unfettered ship. "We surrender. If you land now, you will receive your reward."`
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Anomalocaris"
+			not "unfettered: fled with the relic"
+			"old reputation: Hai (Unfettered)" < -2000
+		personality entering heroic mute marked plunders target unconstrained
+		ship "Shield Beetle (Prototypes)" "Requiem"
+		conversation
+			`You have managed to defeat the Unfettered challenger. Now it's time to land and claim your reward.`
+			action
+				set "unfettered: beaten the bugs"
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			not "unfettered: fled with the relic"
+		ship "Lightning Bug (Unfettered Shipyards)" "Dungeon Master"
+		personality launching marked staying surveillance uninterested
```

Comment:
```suggestion
		personality launching marked staying uninterested
```
Surveillance trumps uninterested - the ship will still scan you first. Just better to leave this out so you aren't visually dogpiled.

Reply:
it doesn't scan much anyway